### DISCUSSION TITLE
Sort folders

### DISCUSSION THREAD

**Discussion by fullcavediver** [Radiokot/photoprism-android-client#151] [Ideas]
Hi @Radiokot,
The folders are a great addition to the app. Thank you! Could you please add a sorting option? I think most people will probably want to sort by date/newest first rather than by name. Thank you very much!

**Comment by Radiokot** [comment]
It would be nice to have

**Reply by wufengzlw** [reply]
Yes
I just scanned some old photo into digital version and added photo to the prism
it sorted photo based on the scanning time, even when I have the naming of photos.

**Reply by Radiokot** [reply]
In the gallery view of the app, photos are always ordered by creation date, from newest to oldest. To get a correct order, use some batch EXIF edit program to set corresponding attributes in files and then re-index.

**Comment by fullcavediver** [comment]
Sorting photos by date is not the problem - I would like to sort the folders. I now have over 30,000 photos from the last 30 years in 650 folders and the first one from the 90s is listed first. Maybe sorting by name but in descending order - from 'z' to 'a'

**Reply by Radiokot** [reply]
Understood

**Comment by Radiokot** [comment]
Now it is possible to change folder sorting
https://github.com/Radiokot/photoprism-android-client/releases/tag/1.31.0(46)

**Reply by ippocratis** [reply]
Nice addition.
Cheers

**Comment by fullcavediver** [comment]
Many many thanks!

Am 28. September 2024 11:34:05 MESZ schrieb Oleg Koretsky ***@***.***>:
